moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -567,6 +567,17 @@ void TrajectoryExecutionManager::reloadControllerInformation()
       known_controllers_[ci.name_] = ci;
     }
 
+    names.clear();
+    controller_manager_->getActiveControllers(names);
+    for (std::map<std::string, ControllerInformation>::iterator it = known_controllers_.begin();
```

Comment:
I think with new code we should aim to use range-based loops or std algorithms. Also, is there a chance that `getActiveControllers()` returns names for which we don't have a known controller? If no, iterating names and accessing the map directly would make more sense to me